packages/flutter_test/lib/src/binding.dart

```diff
@@ -517,14 +517,24 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   /// test failures.
   bool showAppDumpInErrors = false;
 
-  /// Call the testBody inside a [FakeAsync] scope on which [pump] can
-  /// advance time.
+  /// Runs `setUpBody` with the configuration specified in the binding.
+  ///
+  /// Returns a future which completes when the setup has run.
+  ///
+  /// [runSetup] and [runTest] should execute code in the same [Zone] to
```

Comment:
nit (here and below): doc comments should be proper sentences and start with a capital letter.

Reply:
What would you prefer here:

/// Functions [runSetup] and [runTest] should .....

?